modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -202,14 +203,14 @@ NOTE: Using `text_patterns: []` to turn `text_patterns` off is not supported. It
 
 After upgrading:
 
-* rename any options in your {productname} init configuration to match the new name.
-* remove `textpattern` from your plugins list.
+* Rename any options in your {productname} init configuration to match the new name.
+* Remove `textpattern` from your plugins list.
 
 === `tinymce.activeEditor.options.get`
```

Comment:
```suggestion
=== `plugins` option
```
Maybe to match the changes @metricjs made in the other PR?

Reply:
It needs to be a sub-heading of plugins, too.